### PR TITLE
Edit Istio docs

### PR DESCRIPTION
**Pull request by catherineluse**
I will use this PR to update the Istio docs according to feedback as it comes in

### CODE REVIEW

File: content/rancher/v2.x/en/cluster-admin/tools/istio/rbac/_index.md [hunk 1]
```diff
@@ -51,6 +51,6 @@ To configure who has permission to access the Kiali and Jaeger UI,
 | Enable and disable Istio for a namespace | ✓              | ✓              | ✓              |                           |
 | Enable and disable Istio on workloads    | ✓              | ✓              | ✓              |                           |
 | Configure Istio with `kubectl`           | ✓              | ✓              | ✓              | ✓                         |
-| View Prometheus UI and Grafana UI           | ✓              | ✓              | ✓              | ✓                         |
-| View Kiali UI and Jaeger UI  | ✓              |                |                |                           |
+| View Prometheus UI and Grafana UI           | ✓              |               |               |                          |
+| View Kiali UI and Jaeger UI ([Configurable](#access-to-visualizations)) | ✓              |                |                |                           |
 | View Istio project dashboard, traffic graph and traffic metrics | ✓          | ✓              | ✓            | ✓           |
```
Comment (bmdepesa) [inline content/rancher/v2.x/en/cluster-admin/tools/istio/rbac/_index.md]: By default, only the cluster owner will see the traffic graph. Project members will only a subset of traffic metrics, and cannot see the traffic graph as that comes from Kiali.

File: content/rancher/v2.x/en/cluster-admin/tools/istio/rbac/_index.md [hunk 2]
```diff
@@ -51,6 +51,6 @@ To configure who has permission to access the Kiali and Jaeger UI,
 | Enable and disable Istio for a namespace | ✓              | ✓              | ✓              |                           |
 | Enable and disable Istio on workloads    | ✓              | ✓              | ✓              |                           |
 | Configure Istio with `kubectl`           | ✓              | ✓              | ✓              | ✓                         |
```
Comment (bmdepesa) [inline content/rancher/v2.x/en/cluster-admin/tools/istio/rbac/_index.md]: Read only members will not be able to edit anything with `kubectl`

File: content/rancher/v2.x/en/cluster-admin/tools/istio/rbac/_index.md (hunk 1 above)
Comment (catherineluse) [reply content/rancher/v2.x/en/cluster-admin/tools/istio/rbac/_index.md]: edited the table and added an asterisk with basically this note

File: content/rancher/v2.x/en/cluster-admin/tools/istio/rbac/_index.md (hunk 2 above)
Comment (catherineluse) [reply content/rancher/v2.x/en/cluster-admin/tools/istio/rbac/_index.md]: removed check mark

File: content/rancher/v2.x/en/cluster-admin/tools/istio/rbac/_index.md [hunk 3]
```diff
@@ -20,7 +20,7 @@ After Istio is enabled in a cluster, project owners and members have permission
 - Enable and disable Istio sidecar auto-injection for namespaces
 - Add the Istio sidecar to workloads
 - View the traffic metrics and traffic graph for the cluster
-- View the Prometheus UI and the Grafana UI
+- View the Kiali and Jaeger visualizations if cluster administrators give access to projet members
```
Comment (bmdepesa) [inline content/rancher/v2.x/en/cluster-admin/tools/istio/rbac/_index.md]: project* members

**Pull request merged**